src/battle/partner/goombario.c

```diff
@@ -149,7 +149,7 @@ ApiStatus func_80238E04_6F1EE4(Evt* script, s32 isInitialCall) {
 ApiStatus func_80238E48_6F1F28(Evt* script, s32 isInitialCall) {
     EffectInstance* effect = D_8023CDA0;
 
-    effect->data->pos.y = 144.0f;
+    ((EffectInstanceData*)effect->data)->pos.y = 144.0f;
     effect->flags |= 0x10;
```

Comment:
Not entirely sure if there are flags for Effects of EffectInstances, but if there are, please add them here